google/cloud/forseti/common/gcp_api/securitycenter.py

```diff
@@ -154,23 +151,3 @@ def create_finding(self, finding, organization_id=None, source_id=None,
                 violation_data = (
                     finding.get('source_properties').get('violation_data'))
                 raise api_errors.ApiExecutionError(violation_data, e)
-
```

Comment:
ditto: do we still need the `if` check above?  Can we rely on this as the default behavior?  (I think so.)